src/Unidevel.OpenWeather/OpenWeatherClient.cs

```diff
@@ -9,34 +9,46 @@ public class OpenWeatherClient : IDisposable, IOpenWeatherClient
     {
         private readonly IConfiguration _configuration;
         private readonly string _apiKey;
+        private bool _disposed = false;
+
+        private readonly HttpClient _httpClient = new HttpClient();
```

Comment:
That's partially fine, however it makes this implementation vulnerable to multithreading issues. Can you use 'using' clause where you need httpClient only? Just like previous webClient was used. Or explain why you prefer different implementation.